### DISCUSSION TITLE
How to deal with strict-origin-when-cross-origin?

### DISCUSSION THREAD

**Discussion by changsin** [mitmproxy/mitmproxy#6845] [General]
When I tried accessing sites like https://www.bbc.com/ or https://www.bbc.co.uk/, I got an Access denied error.
Using Chrome developer tools, what I see is:

Referrer Policy:
strict-origin-when-cross-origin

If I put the domains in the ignoreHosts list, I can access them, but is there a way to still monitor the traffic without using IgnoreHosts?

Here is the version info:

Mitmproxy: 10.2.4
Python:    3.10.11
OpenSSL:   OpenSSL 3.2.1 30 Jan 2024
Platform:  macOS-14.4-arm64-arm-64bit

**Comment by mhils** [comment]
I think `strict-origin-when-cross-origin` is unrelated to the error you are seeing. The error you are seeing looks like BBC is using some sort of Web Application Firewall that blocks non-browser clients.

refs https://github.com/mitmproxy/mitmproxy/issues/4575
